src/app/server/Mdns.cpp

```diff
@@ -247,7 +247,7 @@ CHIP_ERROR MdnsServer::GetCommissionableInstanceName(char * buffer, size_t buffe
 /// Set MDNS operational advertisement
 CHIP_ERROR MdnsServer::AdvertiseOperational()
 {
-    for (const Transport::FabricInfo & fabricInfo : Server::GetInstance().GetFabricTable())
+    for (const FabricInfo & fabricInfo : Server::GetInstance().GetFabricTable())
```

Comment:
Could the FabricTable be injected into a given MdnsServer or the calls that need it, rather than accessing a singleton at the outer level of a loop like here and a few other places?

Reply:
Maybe it can be done. MdnsServer itself is a singleton right now. Need to look more carefully on how and from where in the code it could be initialized. Right now it has a static instance, that accesses other singletons from Server class..